juniper/src/http/playground.rs

```diff
@@ -23,7 +23,7 @@ pub fn playground_source(
   <title>GraphQL Playground</title>
   <link rel="stylesheet" href="//cdn.jsdelivr.net/npm/graphql-playground-react/build/static/css/index.css" />
   <link rel="shortcut icon" href="//cdn.jsdelivr.net/npm/graphql-playground-react/build/favicon.png" />
-  <script src="//cdn.jsdelivr.net/npm/graphql-playground-react@1.7.27/build/static/js/middleware.js"></script>
+  <script src="//cdn.jsdelivr.net/npm/graphql-playground-react/build/static/js/middleware.js"></script>
```

Comment:
I don't think that removing the version at all is a good way to go, as it's the same as have `tokio = "*"` in `Cargo.toml`. You will never now when the things blew up on breaking change.

But!

We can just follow semver and specify `@1` version. So we catch up with recent updates but don't hit major version breakage.